docs/lib/debugging/menu.json

```diff
@@ -0,0 +1,4 @@
+{
+  "title": "Debugging",
+  "items": ["dev-menu"]
```

Comment:
do we plan on more debugging content? could we scaffold this into the `utilities` section?

Reply:
@TrekSoft asked me to create a debugging section and envisioned adding logging information in the future, although I see now that Amplify JS included logging information in the utilities section.

Reply:
Last minute pitch here: `utilities` might be better because we could include information about the logging category, `Hub`, other tooling. @TrekSoft any thoughts here before we ship it?